README.md

```diff
@@ -166,6 +166,27 @@ Note that the following will fail:
 Use the IsNil checker below for this kind of nil check.
 
 
+### ErrorAs
```

Comment:
These docs are great, but as already mentioned at https://github.com/frankban/quicktest/pull/111 the README is generated, and we should update doc.go instead.

Reply:
:+1: